app/controllers/hyrax/depositors_controller.rb

```diff
@@ -15,14 +14,18 @@ def index
     end
 
     def create
-      grantor = authorize_and_return_grantor
-      grantee = ::User.from_url_component(params[:grantee_id])
-      if grantor.can_receive_deposits_from.include?(grantee)
-        head :ok
+      if (params[:user_id].sub '.', '-dot-') == (params[:grantee_id].sub '.', '-dot-')
+        render json: { name: "FAILURE" }
```

Comment:
Maybe it would be good to return a different http status code here since it is a failure?

Reply:
@cjcolvar I will try status: :bad_request